### ISSUE TITLE
Ambiguous licensing

### ISSUE THREAD

**Issue by booty-bumping** [Lordmau5/FFS#30]
From the curseforge description:

> Modpack policy: One can use the mod in their modpacks, as long as they are crediting us properly for it.
> As well, monetization is not allowed.

The MIT license allows unlimited commercial use. My recommendation would be either to just reword this to be more of a suggestion ("If you own a mod repost website, please don't post this mod") or to switch to a license that is compatible with that requirement such as [CC BY-NC 4.0](https://creativecommons.org/licenses/by-nc/4.0/legalcode.txt). Just know that a non-commercial license is no longer technically [open source](https://opensource.org/osd-annotated). But I don't think it's much of a problem for a minecraft mod, as long as you have all of the other freedoms.

**Comment by Lordmau5** [comment]
Thank you for bringing this up - I'm relatively sure I'll be switching it over to All-Rights-Reserved, seeing as that's what we use for another mod as well and ... it works perfectly fine over there.

**Comment by gigabit101** [comment]
Just to get this out the way as I know how this goes.
I consent to Lordmau5 changing the license and give full permission to continue using any code I wrote for part of this mod

GIgabit101

**Comment by Lordmau5** [comment]
Ah yes, almost forgot about that.

That means @modmuss50 also needs to consent to the license-change.

As for the old 1.10 code / FFS v1... that has more contributors but is also a dead codebase at this point, so it's gonna stay on MIT.

**Comment by booty-bumping** [comment]
Do what you wish but I think all rights reserved is overly strict for a Minecraft mod. CC BY-NC pretty much covers all of the bad actors in the Minecraft community, while still having all of the open source freedoms (derivatives, non-monetized modpack distribution, access to source code) that are good for the community.

There's also sharealike variants of all the CC licenses. This comes with the additional restriction that any forks, unless they are given a special exception, have to release under the same license. So if you release a fork to the public, you have to give back any code improvements under a license compatible with the upstream.

>As for the old 1.10 code / FFS v1... that has more contributors but is also a dead codebase at this point, so it's gonna stay on MIT.

Well, any old code is going to stay on MIT anyways, since MIT is unrevokable. Only once the code is changed the license is revoked.

**Comment by Lordmau5** [comment]
CC BY-NC doesn't sound bad either, so maybe considering that instead - @gigabit101 you still fine with consent in this regard?

In that case also asking @modmuss50 if he'd be fine with CC BY-NC instead of All Rights Reserved (and potentially also feedback in regards to which one would be a better fit)

**Comment by gigabit101** [comment]
Yeah, that's fine with me, you have full permissions to use my code in this mod  with any license you wish
:)

**Comment by modmuss50** [comment]
Yes. You can change the license as you wish for my contributions.

**Comment by Lordmau5** [comment]
Switched to CC BY-NC with this commit https://github.com/Lordmau5/FFS/commit/bb7d68ef22217b40c7761e23c058a26528c93c04
